util/SqsMessageHandler.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Reads SQS messages and returns the message if available
+ * @param {Object} params Queue URL and other SQS attributes
+ * @param {Object} sqs AWS SQS service object
+ * @returns {Promise} Returns false if no message was recieved 
+ * or if there was an error, otherwise returns the recieved message
+ */
+function readMessageFromSqs(params, sqs) {
```

Comment:
function `pushes p`. get it? because it's a queue? you push stuff to the queue. haha.

anyways, what if you wrap the stuff in a try catch for length safety like:
```js
function readMessageFromSqs(params, sqs) {
  return new Promise((resolve) => {
    try {
      // ...
    } catch(e) {
      resolve(false);
    }
  })
```